playbooks/tests/tasks/integration.yml

```diff
@@ -125,3 +112,74 @@
            cirros@{{ floating_ip.floating_ip.floating_ip_address }}
            ping -c 5 www.google.com
   when: floating_ip.floating_ip.floating_ip_address is defined
+
+- name: nova can attach cinder volume
+  environment:
+    PYTHONPATH: "{{ basevenv_lib_dir|default(omit) }}"
+  os_server_volume:
+    auth:
+      auth_url: "{{ endpoints.auth_uri }}"
+      project_name: admin
+      username: admin
+      password: "{{ secrets.admin_password }}"
+    server: turtle-stack
+    volume: turtle-vol
+  when: cinder.enabled|bool
+
+- name: write volume in VM
+  shell: ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no
+         -i /root/.ssh/id_rsa
+         cirros@{{ floating_ip.floating_ip.floating_ip_address }}
+         sudo dd if=/dev/zero of=/dev/vdb count=100 bs=1M
```

Comment:
As we write disk directly, there is no page cache. So we are good even no `sync` after `dd`.